india_stocks_api/internal/fivepaisa/api/order_api.py

```
@@ -0,0 +1,627 @@
+        
+        logger.info(f"Order Response: {response_data}")
+        
+        if response_data['head']['statusDescription'] == "Success":
+            orderid = response_data['body']['BrokerOrderID']
+        else:
+            orderid = None
+            
+        # Add status attribute to make it compatible with place_order.py
+        response.status = response.status_code
+            
+        return response, response_data, orderid
```

Ruff (0.14.14)

Consider moving this statement to an `else` block

(TRY300)